home/web/client/error_pages.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Roboto";
+  src: url('~roboto-font/fonts/Roboto/roboto-light-webfont.eot');
+  src: url('~roboto-font/fonts/Roboto/roboto-light-webfont.eot?#iefix') format('embedded-opentype'), url('~roboto-font/fonts/Roboto/roboto-light-webfont.woff') format('woff'), url('~roboto-font/fonts/Roboto/roboto-light-webfont.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto Mono';
+    src: url('~roboto-mono-webfont/fonts/RobotoMono-Light.ttf') format('truetype');
+    font-style: normal;
+    font-weight: 300;
+    text-rendering: optimizeLegibility;
+}
+
+html:root, :root body {
```

Comment:
Wouldn't it be a no-op?
https://developer.mozilla.org/en-US/docs/Web/CSS/:root

Seems that `:root` is `html`, so there will be no descendant `html` inside `:root`.